### PR TITLE
Fix login delay issue

### PR DESCRIPTION
**Pull request by mathiascode**
We sent a message to the server before we were logged in, which delayed the login process.

### CODE REVIEW

**Comment by mathiascode** [comment]
This CI issue is becoming a bit ridiculous. I'll look into it.

**Comment by kiplingw** [comment]
Indeed. Thanks @mathiascode.

**Comment by mathiascode** [comment]
Seems to be caused by the recent release of MySQL 5.7.31. I don't think it's necessary to manually invoke an upgrade, the Ubuntu images seem to be refreshed every now and then.

**Comment by kiplingw** [comment]
Some of the packages I noticed are pulled from PPAs the service is manually adding.

File: .github/workflows/autopkgtest.yml [hunk 1]
```diff
@@ -76,8 +76,6 @@ jobs:
           name: debuild-source
       - name: Install autopkgtest dependencies
         run: |
-          sudo apt update
```
Comment (kiplingw) [inline .github/workflows/autopkgtest.yml:79]: This is definitely not a good idea and is a security risk.

**Comment by mathiascode** [comment]
@kiplingw merge?

**Comment by kiplingw** [comment]
Almost. See change requested above.

**Comment by mathiascode** [comment]
The only change request I see is the CI-related one, which was moved to another PR and merged.

**Comment by kiplingw** [comment]
Sorry, forgot to submit review. Reload.

File: debian/changelog [hunk 2]
```diff
@@ -1,8 +1,8 @@
-nicotine (2.0.2-dev1) focal; urgency=medium
+nicotine (2.1.0-dev1) focal; urgency=medium
 
   * Development version.
 
- -- Mathias <mail@mathias.is>  Mon, 19 Jul 2020 13:58:00 +0300
+ -- Mathias <mail@mathias.is>  Mon, 30 Jul 2020 20:32:00 +0300
```
[the remark this answers is not in the document]
Comment (mathiascode) [reply debian/changelog]: Silly mistake. Setting up a proper environment for building Debian packages now.

**Comment by mathiascode** [comment]
No worries.

**Pull request merged**